.gitignore

```diff
@@ -114,3 +114,6 @@ CMakeFiles/TargetDirectories.txt
 # clangd
 .cache
 thirdparty/zlib-1.2.11/
+
+.env
```

Comment:
Can you add a `example.env` file to the repo to show how this should be configured?

Reply:
I have a version of this in #36 but I think your PR is more of a complete solution.

Reply:
Yes, I will do this later. Currently, I am just up to get this running. My current struggle is to have a MasterServer binary which actually works xD

Reply:
Haha yeah I get this is a draft! Just so excited by this solution already! Lemme know if you want any help :)